packages/stream-tile-handler/src/tile-document-handler.ts

```diff
@@ -169,14 +170,15 @@ export class TileDocumentHandler implements StreamHandler<TileDocument> {
     /**
      * Verifies commit signature
      * @param commit - Commit to be verified
+     * @param meta - Commit metadata
      * @param context - Ceramic context
      * @param did - DID value
      * @private
      */
-    async _verifySignature(commit: any, context: Context, did: string): Promise<DIDResolutionResult> {
+    async _verifySignature(commit: any, meta: CommitMeta, context: Context, did: string): Promise<DIDResolutionResult> {
         let result
         try {
-            result = await context.did.verifyJWS(commit)
+            result = await context.did.verifyJWS(commit, {atTime: meta.timestamp})
```

Comment:
wow, all that for this one magical line.  Amazing

Reply:
😆

Reply:
There was a PR in JWS to support this fwiw.